locales/fr.json

```diff
@@ -0,0 +1,27 @@
+{
+  "Feedback": "Commentaires",
+  "Close": "Fermer",
+  "Share your thoughts and insights to help us achieve an exceptional user experience.": "Partagez vos réflexions et suggestions pour nous aider à offrir une expérience utilisateur exceptionnelle.",
+  "Report an issue": "Signaler un problème",
+  "Suggest an improvement": "Suggérer une amélioration",
+  "Fix a typo": "Corriger une faute de frappe",
+  "Ask a question": "Poser une question",
+  "Other": "Autre",
+  "Intent": "Intention",
+  "Description": "Description",
+  "Screenshot": "Capture d'écran",
+  "optional": "facultatif",
+  "Capture screenshot": "Capturer l'écran",
+  "Preview": "Aperçu",
+  "Remove": "Supprimer",
+  "Try again": "Réessayer",
+  "Failed to capture screenshot.": "Échec de la capture d'écran.",
+  "Email": "E-mail",
```

Comment:
Email would be translated by "Courriel" in french. It is not recommended to use "e-mail" because it is an anglicism ([src](https://www.larousse.fr/dictionnaires/francais/e-mail/28496))

Reply:
@meliecool I added a fr-CA version